crates/fig_os_shim/src/process_info/windows.rs

```diff
@@ -1,9 +1,15 @@
 use std::path::PathBuf;
 use std::sync::Weak;
 
-use sysinfo::{ProcessesToUpdate, System};
-
-use super::pid::{Pid, RawPid};
+use sysinfo::{
```

Comment:
The changes in this diff were produced by executing `cargo +nightly fmt`.
It includes no functional modifications.